libraries/chain/controller.cpp

```diff
@@ -658,27 +701,30 @@ struct building_block {
                };
             },
             [&](building_block_if& bb) -> assembled_block {
-               // compute the action_mroot and transaction_mroot
-               auto [transaction_mroot, action_mroot] = std::visit(
-                  overloaded{[&](digests_t& trx_receipts) { // calculate the two merkle roots in separate threads
+               // compute the transaction_mroot
+               auto transaction_mroot = std::visit(
+                  overloaded{[&](digests_t& trx_receipts) { // calculate the merkle root in a thread
```

Comment:
Is there a reason why the action_mroot calculation was removed from here? Is it calculated elsewhere now?

Reply:
Will revert back. Thanks for pointing this out!